uiplib/scrape.py

```diff
@@ -36,23 +36,27 @@ def get_images(url):
     '''
     scrape images from /r/wallpapers
     '''
-    soup = make_soup(url)
-    # this makes a list of bs4 element tags
-    thumbnails = soup.find_all("a", class_="thumbnail", href=True)
+    image_links = []
+    p_url = url
+    while len(image_links) < NUMBER_OF_IMAGES_TO_PARSE:
```

Comment:
@teja-315 what if the entire reddit.com/r/wallpapers doesnt have that many images? i guess you should